iothub/service/tests/MessageTests.cs

```diff
@@ -1,105 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
```

Comment:
this class has been moved to Messaging/MessageTests.cs

Reply:
Where in this PR?

Reply:
I see, the tests were already in that file, yes?

Reply:
Yup!